site/zh/blog/new-in-devtools-99/index.md

```diff
@@ -31,165 +30,211 @@ draft: true
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/1e73eb62955de7c4b0920575c7b374d47dab6a65 #}
 
-Chromium issue: [212374](https://crbug.com/212374)
+Chromium 议题: [212374](https://crbug.com/212374)
 
 
 <!-- ### Improved source folder tree in the Sources panel {: #source-tree } -->
+### 优化 Sources 面板里面源文件夹树的显示问题 {: #source-tree }
 
 <!-- The source folder tree in the **Sources** panel is now improved with less clutter in the folder structures and naming (e.g. “../”, “./”, etc). Under the hood, this is the result of normalizing the absolute source URLs in the sourcemaps. -->
+现在，**Sources** 面板里面源文件夹树变得更加简洁，并且命名更加易读（例如：“../”、“./”等）。底层是通过规范化 sourcemaps 里面的绝对源 URL 来实现的。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Wl1pPVfQ51NaCtpp3KuY.png", alt="Improved source folder tree in the Sources panel", width="800", height="444" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Wl1pPVfQ51NaCtpp3KuY.png", alt="优化 Sources 面板里面源文件夹树的显示", width="800", height="444" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/68613ab02f6d556a2c5ac68ea08f466a534c6bd9 #}
 
-Chromium issue: [1284737](https://crbug.com/1284737)
+Chromium 议题: [1284737](https://crbug.com/1284737)
 
 
 <!-- ### Display worker source files in the Sources panel {: #worker-sourcemap } -->
+### Sources 面板里面显示 Worker 源文件 {: #worker-sourcemap }
 
 <!-- [Worker](https://web.dev/workers-overview/) (e.g. web worker, service worker) source files with relative SourceURL are now displayed in the **Source** panel. Previously, worker source files were not handled correctly. -->
+现在，[Worker](https://web.dev/workers-overview/)（例如：web worker、service worker）源文件在使用相对 SourceURL 的情况下，会被显示在 **Source** 面板里面。在此之前，Worker 源文件不会被正确处理。
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/apH5n92bqYWINMQn5VXa.png", alt="ALT_TEXT_HERE", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/apH5n92bqYWINMQn5VXa.png", alt="Sources 面板里面显示 Worker 源文件", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/6e877d5e1a3ccca22e866fb2a70330613aeb6964 #}
 
-Chromium issue: [1277002](https://crbug.com/1277002) 
+Chromium 议题: [1277002](https://crbug.com/1277002)
 
 
 <!-- # Chrome’s Auto Dark Theme updates {: #auto-dark-mode } -->
+## Chrome 自动深色模式的更新 {: #auto-dark-mode }
 
 <!-- The [Auto Dark Theme emulation](/blog/new-in-devtools-96/#auto-dark-mode) UI is now simplified. It is a checkbox now, it was a dropdown previously. -->
+现在，[自动深色模式模拟](/blog/new-in-devtools-96/#auto-dark-mode) UI 变得更加简洁。当下是个复选框，之前是个下拉框。
 
 <!-- Apart from that, when the [Auto Dark Theme](/blog/auto-dark-theme/) is enabled, the **Emulate perfers-color-scheme** dropdown will be disabled and set to **perfers-color-scheme: dark** automatically. -->
+除了上面说的这些，在开启[自动深色主题](/blog/auto-dark-theme/)功能之后，**Emulate perfers-color-scheme** 下拉框会被禁用掉，并且下拉框选项还会被自动设置为 **perfers-color-scheme: dark**。
```

Comment:
@chybie I think there's a typo here: `perfers-color-scheme` should be `prefers-color-scheme`?

Reply:
Updated